### PR TITLE
REL-2550 SonarQube 9.9 LTS

### PR DESCRIPTION
**Pull request by christophelevis**

### CODE REVIEW

**Comment by christophelevis** [comment]
The availability for the M1 Docker image is still to be confirmed.

File: update-center-source.properties [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-devVersion=9.9-SNAPSHOT
-publicVersions=8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,8.9.10,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5,9.6,9.6.1,9.7,9.7.1,9.8
+devVersion=10.0-SNAPSHOT
+publicVersions=8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,8.9.10,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5,9.6,9.6.1,9.7,9.7.1,9.8,9.9
 privateVersions=4.2,4.3,4.3.1,4.3.2,4.3.3,4.4,4.4.1,4.5,4.5.1,4.5.2,4.5.4,4.5.5,4.5.6,4.5.7,5.0,5.0.1,5.1,5.1.1,5.1.2,5.2,5.3,5.4,5.5,5.6,5.6.1,5.6.2,5.6.3,5.6.4,5.6.5,5.6.6,5.6.7,6.0,6.1,6.2,6.3,6.3.1,6.4,6.5,6.6,6.7,6.7.1,6.7.2,6.7.3,6.7.4,6.7.5,6.7.6,6.7.7,7.0,7.1,7.2,7.2.1,7.3,7.4,7.5,7.6,7.7,7.8,7.9,7.9.1,7.9.2,7.9.3,7.9.4,7.9.5,7.9.6,8.0,8.1,8.2,8.3,8.3.1,8.4,8.4.1,8.4.2,8.5,8.5.1,8.6,8.6.1,8.7,8.7.1,8.8
 
-ltsVersion=8.9.10
+ltsVersion=9.9
+
+9.9.description=Long-Term Support version - Faster analysis for DotNet, PHP and Python PRs, Docker security rules, cloud secret detection rules, support for Apple Silicon M1, hardening for long-term reliability, bug fixes and improvements
```
Comment (ganncamp) [inline update-center-source.properties:10]: * Thought it was supposed to be faster analysis for PRs of all langauges?
* It _is_ faster full analysis for Java
* IMO drop "rules" for just: "Docker security, cloud secret detection"
* I'd specify "Docker deployment on Apple Silicon M1"
* I'd drop "bug fixes and improvements"

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: update-center-source.properties (hunk 1 above)
Comment (christophelevis) [reply update-center-source.properties:10]: I listed here only the features added by 9.9.

Comment (christophelevis) [reply update-center-source.properties]: I made an update based on your feedback.

File: update-center-source.properties [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
-devVersion=9.9-SNAPSHOT
-publicVersions=8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,8.9.10,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5,9.6,9.6.1,9.7,9.7.1,9.8
+devVersion=10.0-SNAPSHOT
+publicVersions=8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,8.9.10,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5,9.6,9.6.1,9.7,9.7.1,9.8,9.9
 privateVersions=4.2,4.3,4.3.1,4.3.2,4.3.3,4.4,4.4.1,4.5,4.5.1,4.5.2,4.5.4,4.5.5,4.5.6,4.5.7,5.0,5.0.1,5.1,5.1.1,5.1.2,5.2,5.3,5.4,5.5,5.6,5.6.1,5.6.2,5.6.3,5.6.4,5.6.5,5.6.6,5.6.7,6.0,6.1,6.2,6.3,6.3.1,6.4,6.5,6.6,6.7,6.7.1,6.7.2,6.7.3,6.7.4,6.7.5,6.7.6,6.7.7,7.0,7.1,7.2,7.2.1,7.3,7.4,7.5,7.6,7.7,7.8,7.9,7.9.1,7.9.2,7.9.3,7.9.4,7.9.5,7.9.6,8.0,8.1,8.2,8.3,8.3.1,8.4,8.4.1,8.4.2,8.5,8.5.1,8.6,8.6.1,8.7,8.7.1,8.8
 
-ltsVersion=8.9.10
+ltsVersion=9.9
+
+9.9.description=Long-Term Support version - Faster analysis for DotNet, PHP and Python PRs, Docker security and cloud secret detection rules, Docker deployment on Apple Silicon M1, hardening for long-term reliability
```
Comment (ganncamp) [inline update-center-source.properties:10]: I suppose it's a philosophy question. IMO it should promote the LTS or, I guess, be super minimal
8.9 description:
> Long-Term Support version - Hardening for long-term reliability, bug fixes and small improvements

**Review by ganncamp — APPROVED** [review]

**Pull request merged**